manifests/native.json

```diff
@@ -1039,6 +1039,94 @@
       "replacement": "Array.prototype.lastIndexOf",
       "mdnPath": "Global_Objects/Array/lastIndexOf",
       "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "abort-controller",
+      "nodeVersion": "0.10.0",
+      "replacement": "AbortController",
+      "mdnPath": "Global_Objects/AbortController",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "array.prototype.findlast",
+      "nodeVersion": "0.10.0",
```

Comment:
The Node version is incorrect for most of those packages. However, I agree that they should be replaced everywhere.

Reply:
good catch :) lazy copy/paste work on my side, will fix

Reply:
fixed